CHANGELOG

```diff
@@ -51,7 +51,7 @@ Data handling
 * Fix part of #5914: Complete Takeout Frontend (#9424)
 * Fix part of #5914: Add JSON rendering testing to Takeout (#9301)
 * Fixed user updates. (#9252)
-*  Added update username functionality in the admin tab. (#9191)
+* Added update username functionality in the admin tab. (#9191)
```

Comment:
Thanks for fixing it though it was not that required! :p

Can you stop the build and ask Sean for a direct merge?

Reply:
Sure, it is failing anyways after two restarts.

@seanlip can you please merge this one - tests aren't required here.